### PR TITLE
Feature#13 setup devise cancan rolify

### PR DESCRIPTION
**Pull request by LDVera**
# Description

the basic setup and  configuration of the three gems was created 

Gems: 
- Devise
- Cancancan
- Rolify

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -57,4 +57,6 @@ gem 'devise-jwt'
 gem 'kaminari'
 gem 'rqrcode'
 gem 'cancancan'
```
Comment (MiguelOcegueraM) [inline Gemfile:59]: good choice with this gem!

**Review by MiguelOcegueraM — APPROVED** [review]
The code is very clean, i like it!

File: Gemfile (hunk 1 above)
Comment (jnegrete15) [reply Gemfile:59]: Agree!, cancancan gem seems good option to manage roles

**Review by jnegrete15 — DISMISSED** [review]

**Review by OchoaHub — DISMISSED** [review]

**Review by fego-dev — DISMISSED** [review]
Good Job.

**Review by n0b0dy-su — APPROVED** [review]
nice work, a good way to manage roles

**Review by LPMendoza — APPROVED** [review]

**Review by jnegrete15 — APPROVED** [review]

**Pull request merged**